src/test/java/de/rabea/Game/BoardTest.java

```diff
@@ -75,23 +77,172 @@ public void checkIf9IsAnAvailablePosition() {
 
     @Test
     public void returnsAnArrayOfAllEmptyPosition() {
-        board.placeMark(1, X);
-        board.placeMark(2, O);
-        board.placeMark(3, X);
-        board.placeMark(6, O);
+        board.placeMark(1, Mark.X);
+        board.placeMark(2, Mark.O);
+        board.placeMark(3, Mark.X);
+        board.placeMark(6, Mark.O);
         List<Integer> emptyCells = new ArrayList<Integer>(Arrays.asList(0,4,5,7,8));
         assertEquals(emptyCells, board.emptyCells());
     }
 
+    @Test
+    public void indexOfLastCellInEachRow() {
+        List<Integer> cells = new ArrayList<Integer>(Arrays.asList(2,5,8));
+        assertEquals(cells, board.indexOfLastCellPerRow());
+    }
+
+    @Test
+    public void indexOfLastCellOfBoard() {
+        assertEquals(8, board.indexOfLastCell());
+    }
+
+    @Test
+    public void itKnowsTheWidthOfTheBoard() {
+        assertEquals(3, board.getDimension());
+    }
+
+    @Test
+    public void itGetsAllRowsOfTheBoard() {
+        List<List<Integer>> rows = new ArrayList<List<Integer>>();
+        List<Integer> row1 = new ArrayList<Integer>(Arrays.asList(0,1,2));
+        List<Integer> row2 = new ArrayList<Integer>(Arrays.asList(3,4,5));
+        List<Integer> row3 = new ArrayList<Integer>(Arrays.asList(6,7,8));
+        rows.add(row1);
+        rows.add(row2);
+        rows.add(row3);
+        assertEquals(rows, board.getRows());
+    }
+
+    @Test
+    public void itGetsAllColumnsOfTheBoard() {
+        List<List<Integer>> columns = new ArrayList<List<Integer>>();
+        List<Integer> column1 = new ArrayList<Integer>(Arrays.asList(0,3,6));
+        List<Integer> column2 = new ArrayList<Integer>(Arrays.asList(1,4,7));
+        List<Integer> column3 = new ArrayList<Integer>(Arrays.asList(2,5,8));
+        columns.add(column1);
+        columns.add(column2);
+        columns.add(column3);
+        assertEquals(columns, board.getColumns());
+    }
+
+    @Test
+    public void itGetsTheDiagonals() {
+       List<List<Integer>> diagonals = new ArrayList<List<Integer>>();
```

Comment:
You could define a helper method in this test to construct the columns for you, as it is repeated throughout a few tests.